### PR TITLE
Add linux (ubuntu 21.04) ci workflow

### PR DESCRIPTION
**Pull request by schroedtert**

### CODE REVIEW

**Review by chraibi — COMMENTED** [review]
Do you intend to include VTK8 test?

**Comment by Ozaq** [comment]
@chraibi regarding vtk8, im not sure if we want to do that, right now we say we provide Mac / Windows packages and only support for latest LTS Ubuntu. For all this platforms VTK9 is available without additional effort.

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: jpsvis-unix
```
Comment (Ozaq) [inline .github/workflows/linux.yml:1]: this probably should be `jpsvis-ubuntu`

**Review by Ozaq — COMMENTED** [review]
I suggest to change the config name to jpsvis-ubuntu but other than that it looks fine to me. Also I would not bother with vtk8.

**Review by Ozaq — APPROVED** [review]

**Pull request merged**